### ISSUE TITLE
Error in HTML report for FMRIPREP v21.0.0rc0

### ISSUE THREAD

**Issue by julfou81** [nipreps/fmriprep#2537]
Hi, thank you for the terrific work with FMRIPREP. I tested the new version of FMRIPREP (v21.0.0rc0) and noticed the small bug below in the **Brain mask and (anatomical/temporal) CompCor ROIs** section of the HTML report:
->  the image used as a background for the CompCor ROIs is the non corrected image (visible because the brain is going well out of the brainmask outilne) in v21, whereas it was displaying the corrected image in v20. See example below where the undistorded image is displayed below the outline of the brain mask.
Also,  the **Methods section**, the title for Anatomical data preprocessing is not in **bold**.

![Capture d’écran 2021-09-14 à 09 49 30](https://user-images.githubusercontent.com/32452352/133217683-780b5193-c1d3-4b14-9287-c5857c33ee53.png)

Here is the link to an exemple of such a report:

https://www.dropbox.com/s/79qi3fogiwclh6k/export_report.zip?dl=0

### What version of fMRIPrep are you using?

21.0.0rc0

### What kind of installation are you using? Containers (Singularity, Docker), or "bare-metal"?

Singularity

### What is the exact command-line you used?
```
/usr/local/miniconda/bin/fmriprep --fs-license-file /work/freesurfer/license.txt /work/Phonet /work/Phonet/derivatives --bold2t1w-dof 9 participant --participant-label 08 -w /work/temp_data_Phonet --mem-mb 50000 --omp-nthreads 10 --nthreads 12 --fd-spike-threshold 0.5 --dvars-spike-threshold 2.0 --fs-no-reconall --output-spaces MNI152NLin2009cAsym --ignore slicetiming --return-all-components --debug compcor
```

### Have you checked that your inputs are BIDS valid?

Yes, they are

### Can you find some traces of the error reported in the visual report (at the bottom) or in *crashfiles*?
```
No errors to report!
```

### Are you reusing previously computed results (e.g., FreeSurfer, Anatomical derivatives, work directory of previous run)?

No

**Comment by mgxd** [comment]
Thank you for the report!

This was just corrected in nibabies (nipreps/nibabies@4b0a580b4b2b735fc1c178ab553a6e69357482a1), so should be relatively easy.

**Comment by effigies** [comment]
@mgxd I believe this has been taken care of?

**Comment by mgxd** [comment]
I believe the masking was fixed, so just need to verify boilerplate #2526

**Comment by effigies** [comment]
Okay, since that issue is still open, I'll close this one.
